webpack.config.js

```diff
@@ -6,6 +6,7 @@ const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
 const TerserWebpackPlugin = require('terser-webpack-plugin');
 const CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
```

Comment:
It looks like this is following the package's [Usage with webpack without bundling guidance](https://github.com/mozilla/webextension-polyfill#usage-with-webpack-without-bundling), but I'm not sure this is necessary since we are already bundling; I'd have thought [Basic Setup with module bundlers](https://github.com/mozilla/webextension-polyfill#basic-setup-with-module-bundlers) would be what we'd want to follow (ie, just `import * as browser from 'webextension-polyfill` in chrome-adapter.ts and let webpack deal with bundling like any other package). Did you encounter issues with the usual approach?

Reply:
I tried that first but typings doesn't play nice with that approach... I can give it a second look though

Reply:
Got it working!